index.html

```diff
@@ -78,7 +78,9 @@ <h3>Minerva</h3>
                   Minerva is a GPT-powered Discord bot built 
                   to help students learn software development.
                   <br><br>
-                  Yurij Mikhalevich
+                    <a href="https://github.com/yurijmikhalevich" tartget="_blank">
```

Comment:
Здесь и ниже опечатка в слове `target` :)
Поэтому и не работает.
```suggestion
                    <a href="https://github.com/yurijmikhalevich" target="_blank">
```

Reply:
Давай исправим и можно будет смёржить.

Приложи видео и/или скриншот того, как выглядит приложение и работает фича.